docs/SimpleList.md

```diff
@@ -72,7 +72,14 @@ export const PostList = () => (
 );
 ```
 
-Setting the `linkType` prop to `false` (boolean, not string) removes the link in all list items.
+
+`linkType` accepts the following options:
+
+* `linkType="edit"`: links to the Edit page. This is the default behavior.
+* `linkType="show"`: links to the show page.
+* `linkType={false}`: disables clicking the item.
```

Comment:
I think it is better to say that It removes the link, instead of saying, it disables it